translate/cloud-client/src/test/java/com/example/translate/HybridGlossariesIT.java

```diff
@@ -0,0 +1,154 @@
+import com.google.cloud.translate.v3beta1.ListGlossariesResponse;
+import com.google.cloud.translate.v3beta1.LocationName;
+import com.google.cloud.translate.v3beta1.TranslationServiceClient;
+import com.google.cloud.translate.v3beta1.TranslationServiceClient.ListGlossariesPagedResponse;
+
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.PrintStream;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/**
+ * Tests for HybridGlossaries sample.
+ */
+@RunWith(JUnit4.class)
+@SuppressWarnings("checkstyle:abbreviationaswordinname")
+public class HybridGlossariesIT {
+
+  private static String PROJECT_ID = System.getenv("GOOGLE_CLOUD_PROJECT");
+  private static String OUTPUT_FILE = "output.mp3";
+  private static String INPUT_FILE = "resources/test.jpeg";
+  private static String TEXT_FILE = "resources/test.txt";
+  private static List<String> LANGUAGES;
+  private static String SRC_LANG = "fr";
+  private static String TGT_LANG = "en";
+  private static String GLOSSARY_NAME;
+  private static String GLOSSARY_URI =
+      "gs://cloud-samples-data/translation/bistro_glossary.csv";
+
+  private ByteArrayOutputStream bout;
+  private PrintStream out;
+  private File outputFile;
+
+  @Before
+  public void setUp() {
+    GLOSSARY_NAME = String.format("bistro-glossary-%s", UUID.randomUUID());
+    LANGUAGES = new ArrayList<>();
+    LANGUAGES.add(SRC_LANG);
+    LANGUAGES.add(TGT_LANG);
+    // collect project ID from environment
+    bout = new ByteArrayOutputStream();
+    out = new PrintStream(bout);
+    System.setOut(out);
+  }
+
+  @After
+  public void tearDown() {
+    try (TranslationServiceClient translationServiceClient = TranslationServiceClient.create()) {
+      LocationName locationName =
+              LocationName.newBuilder().setProject(PROJECT_ID).setLocation("us-central1").build();
+
+      ListGlossariesRequest listGlossariesRequest =
+              ListGlossariesRequest.newBuilder().setParent(locationName.toString()).build();
+
+      ListGlossariesPagedResponse listGlossariesPagedResponse =
+              translationServiceClient.listGlossaries(listGlossariesRequest);
+
+      ListGlossariesResponse listGlossariesResponse =
+              listGlossariesPagedResponse.getPage().getResponse();
+
+      for (Glossary glossary : listGlossariesResponse.getGlossariesList()) {
+        DeleteGlossaryRequest request =
+                DeleteGlossaryRequest.newBuilder().setName(glossary.getName()).build();
+
+        translationServiceClient.deleteGlossaryAsync(request).get(300, TimeUnit.SECONDS);
+      }
+    } catch (Exception e) {
+      System.out.format("Failed to delete the glossary.\n%s\n", e.getMessage());
```

Comment:
Don't need to catch the exception here, Junit will handle it. This makes it register as not a test failure.